Mathlib/Analysis/Complex/ValueDistribution/CharacteristicFunction.lean

```diff
@@ -62,4 +62,22 @@ lemma characteristic_sub_characteristic_eq_proximity_sub_proximity (h : Meromorp
     characteristic f ⊤ - characteristic (f · - a₀) ⊤ = proximity f ⊤ - proximity (f · - a₀) ⊤ := by
   simp [← Pi.sub_def, characteristic, logCounting_sub_const h]
 
+/--
+For natural numbers `n`, the counting function counting zeros of `f ^ n` equals `n` times the
+counting function counting zeros of `f`.
+-/
```

Comment:
did you mean `counting` here? This is about the `characteristic` function, which I understand is related.